showcase/src/app/demo-ui/pages/minimap/demo-minimap-page.component.scss

```diff
@@ -0,0 +1,6 @@
+.full-minimap,
+.compact-minimap {
+  .stark-minimap-dropdown-toggle .stark-minimap-dropdown-toggle-menu {
+    right: auto;
```

Comment:
Could we move this code in _packages/stark-ui/src/modules/minimap/components/\_minimap.component.scss_ directly ? 😊